pkg/apis/v1alpha5/suite_test.go

```diff
@@ -521,3 +524,55 @@ var _ = Describe("Limits", func() {
 		Expect(provisioner.Spec.Limits.ExceededBy(provisioner.Status.Resources)).To(MatchError("cpu resource usage of 17 exceeds limit of 16"))
 	})
 })
+
+var _ = Describe("Provisioner Annotation", func() {
+	var provisioner *Provisioner
+	BeforeEach(func() {
+		provisioner = &Provisioner{
```

Comment:
Isn't there a `test.Provisioner()` for this kind of thing?

Reply:
We can't do that here, because we will create an import cycle